### PR TITLE
fixed Console errors

### LINKED ISSUE #3292: Console errors
**Actual Behaviour**

there are various console errors

**Would you like to work on the issue?**

Yes

**Screenshots of the issue**

Screenshot 2020-01-15 at 11 01 56 PM

### PR DESCRIPTION
**Pull request by anishagg17**
Fixes #3292 

Changes: fixed Console errors

Demo Link: https://pr-3293-fossasia-susi-web-chat.surge.sh

### CODE REVIEW

File: src/components/shared/CircleImage/index.js [hunk 1]
```diff
@@ -33,46 +32,16 @@ const Avatar = styled(_Avatar)`
...
-    borderRadius = '100%',
-    size,
-  } = props;
+  const { src, srcset, name, size } = props;
 
-  let backgroundColor = '';
   let innerElement = null;
   if (!src && !srcset) {
-    if (color) {
-      backgroundColor = color;
```
Comment (anishagg17) [inline src/components/shared/CircleImage/index.js]: Color was never passed as a prop

File: src/components/shared/CircleImage/index.js [hunk 2]
```diff
@@ -33,46 +32,16 @@ const Avatar = styled(_Avatar)`
...
-      backgroundColor = colors[i];
-    }
     innerElement = initials(name);
   }
 
   return (
-    <Avatar
-      $backgroundColor={backgroundColor}
-      size={size}
-      $borderRadius={borderRadius}
```
Comment (anishagg17) [inline src/components/shared/CircleImage/index.js]: borderRadius is passed as a prop but never utilized

File: src/components/shared/CircleImage/index.js [hunk 3]
```diff
@@ -33,46 +32,16 @@ const Avatar = styled(_Avatar)`
...
-      // Pick a deterministic color from the list
-      const i = sumChars(name) % colors.length;
-      backgroundColor = colors[i];
-    }
     innerElement = initials(name);
   }
 
   return (
-    <Avatar
-      $backgroundColor={backgroundColor}
```
Comment (anishagg17) [inline src/components/shared/CircleImage/index.js]: This backgroundColor is ineffective

**Comment by akshatnitd** [comment]
@anishagg17 Update the branch,

**Comment by anishagg17** [comment]
Done Sir!

**Pull request merged**